app/src/main/java/edu/usf/sas/pal/muser/util/EventUtils.java

```diff
@@ -0,0 +1,43 @@
+package edu.usf.sas.pal.muser.util;
+
+import android.content.Context;
+
+import com.simplecity.amp_library.ShuttleApplication;
+import com.simplecity.amp_library.model.Song;
+
+import edu.usf.sas.pal.muser.firebase.model.Event;
+import edu.usf.sas.pal.muser.firebase.model.SongData;
+
+
+
+public class EventUtils {
+
+    /**
+     * Function to populate the Event data class.
+     * @param song - The song for which the event occurred.
+     * @param capturedEvent - The event that was captured.
+     * @param context - The context of the Fragment.
+     * @return - Event object
+     */
+    public static Event newEvent(Song song, String capturedEvent, Context context){
+        long currentTimeMS = System.currentTimeMillis();
+        long nanoTime = System.nanoTime();
+        SongData songData = newSongData(song, context);
+        long elapsedTime =  System.currentTimeMillis() - song.startTime;
```

Comment:
This value for `elapsedTime` doesn't seem right to me, given the definition "Elapsed time of the song when the event occurred." If the track has been paused for 30 minutes at the 2m30s mark for a while, and the user hits play, we want to capture `2m30s`. But this would capture 30 minutes? Maybe this variable would be better named something like  `songPosition`, if I'm understanding the definition right?

Reply:
Great find. One way to solve this is to make the `getElapsedTime` function in the `Song` class and check if it gives us the correct results. 

> Maybe this variable would be better named something like songPosition, if I'm understanding the definition right?

Yes, `elapsedTime` does store the song position. I think we should keep the variables as close to the `Song` class to avoid any confusion which may arise while debugging the code? Let me know your thoughts on this.

Reply:
>One way to solve this is to make the getElapsedTime function in the Song class and check if it gives us the correct results.

That sounds good, as long as the data is calculated correctly. 

>I think we should keep the variables as close to the Song class to avoid any confusion which may arise while debugging the code?

Sounds good, if we're deriving this from `Song`